data/json/items/ammo.json

```diff
@@ -884,7 +884,7 @@
     "type": "AMMO",
     "id": "albuterol",
     "category": "drugs",
-    "name": { "str": "albuterol" },
+    "name": { "str_sp": "albuterol" },
```

Comment:
```suggestion
    "name": { "str": "charge of albuterol" },
    "name": { "str_sp": "charges of albuterol" },
```

Reply:
As per the discussion of translator burden, I am disinclined to expand the word.

Reply:
1 "charge"/dose of albuterol is actually 96 micrograms. (My spouse has asthma.)